.github/release.yml

```diff
@@ -0,0 +1,160 @@
+name: Release workflow
+on:
+  workflow_run:
+    workflows: ['Build and test']
+    branches: [main, master, ci-beta, qa-beta, stage-beta, prod-stable, ci-stable, qa-stable, stage-stable, prod-stable]
+    types:
+      - completed
+jobs:
+  pull-scripts:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: mkdir ${{ github.workspace }}/scripts
+      - name: Pull release file
+        run: curl -sSL https://raw.githubusercontent.com/RedHatInsights/insights-frontend-builder-common/master/src/release.sh > ${{ github.workspace }}/scripts/release.sh
+      - run: chmod +x "${GITHUB_WORKSPACE}/scripts/release.sh"
```

Comment:
What's the difference between `${{ github.workspace }}` & `${GITHUB_WORKSPACE}`?

Reply:
We should use the `${{ github.workspace }}` as it is escaped and directly changed to the repo workspace, the other one is ENV variable with same value, but it can be used in places where env variables are needed instead of direct value.